greenhouse_management/models.py

```diff
@@ -79,3 +78,10 @@ class Functionality(models.TextChoices):
         default=Functionality.ACTIVE,
         blank=False,
     )
+    greenhouse = models.ForeignKey(
+        GreenHouse,
+        on_delete=models.CASCADE,
```

Comment:
nie moze byc cascade, bo przy usunieciu device usuniesz greenhouse
mozna poszukac lepszego rozwiazania kombinujac tytaj z on_delete albo w mutacji usuniecia jakos rozlaczyc szklarnie

Reply:
A to nie działa tak że jak mamy ForeignKey z on_delete=models.CASCADE to przy usunięciu device greenhouse nie zostanie usunięty tylko przy usunięciu greenhouse zostaną automatycznie usunięte wszystkie diveces które należały do usuniętego greenhouse?

Reply:
tak, pomieszalo mi sie